packages/bsky/src/api/app/bsky/util/feed.ts

```diff
@@ -12,7 +12,7 @@ export class FeedKeyset extends TimeCidKeyset<FeedRow> {
 }
 
 // For users with sparse feeds, avoid scanning more than one week for a single page
-export const getFeedDateThreshold = (from: string | undefined, days = 1) => {
+export const getFeedDateThreshold = (from: string | undefined, days = 7) => {
```

Comment:
Just double-checking that this number is intended for both timeline and with-friends.

Reply:
yup - optimistic that with these changes, with-friends will be fast as well